etc/fibo-vue/src/views/Courses.vue

```diff
@@ -1,313 +1,96 @@
-<li>Queries: Ultimate parents</li>
-<li>Queries: Counterparty exposure</li>
-<li>&ldquo;Crosswalk&rdquo; data</li>
-</ol>
-</ol>
-
-      </article>
-    </main>
-  </div>
-</template>
-
-<script>
-import helpers from '../store/helpers.js';
-import { outboundClick, outboundLinkClick } from '../helpers/ga';
-
-export default {
-  extends: helpers,
-  name: 'courses',
-  components: {},
-  methods: {
-    outboundClick,
-    outboundLinkClick,
-  },
-};
-</script>
-
-<style lang="scss" scoped>
-</style>
+<template>
+  <div class="container">
+    <main>
+      <article>
+<div>
+<h1>
+<span>Invitation</span>
+</h1>
+<div style="text-align: justify;">
+<p>
+Businesses today need to have comprehensive control and understanding of the data both within the enterprise and from outside. Regulatory pressure in many industries requires that an enterprise be able to find all relevant data about customers, products, business relations and in fact any aspect of the business quickly and accurately. Advances in data analytics have shown that the more meaningful data an enterprise can bring to bear, the more accurately business drivers can be predicted. For this reason, many businesses are migrating from viewing their enterprise information as a collection of data to viewing it as an ongoing graph of meaningful knowledge.
+</p>
+</div>
+<h1>
+<span>Offer</span>
+</h1>
+<div style="text-align: justify;">
+<p>EDMC offers courses in Knowledge Graph, available in multiple forms:</p>
+<ol>
+<li>Instructor-led remote instruction by one of the EDM Council&rsquo;s certified trainers. These courses are open enrollment, registration at the EDM Council web site,</li>
+<li>Enterprise focused instruction. The same course material and instructor as our open enrollment course, but with the audience limited to knowledge workers from your organization. This allows more specific focus on use cases and industries of interest to your needs.</li>
+<li>Formal ELearning courses (to be available summer, 2020). The EDM Council is developing a Certification Package in Knowledge Graph. This is a deep-dive into the full details of knowledge graph technology and business practice.</li>
+</ol>
+<p>In these challanging times EDMC will shortly issue&nbsp;<em>Open Knowledge graph and FIBO Fundamentals Course</em>.</p>
+<h5><span>Open Knowledge Graph and FIBO Fundamentals Course</span></h5>
+<p>This course will be virtually delivered on two consecutive days, 9:00am &ndash; 3:00pm each day&nbsp;with breaks.</p>
+<p>It will consists of 5 main modules:</p>
+<ul>
+<li>A. Knowledge Graph Business Case and Market Adoption [2:15]
+<ul>
+<li>A1. Knowledge Graph Perspective On Data</li>
+<li>A2. Knowledge Graph Industry Use Cases</li>
+<li>A3. Knowledge Graphs Takeaways</li>
+</ul>
+</li>
+<li>B. Knowledge Graph Technical Framework [3:00]
+<ul>
+<li>B1. Knowledge Graph Enterprise Architecture Overview</li>
+<li>B2B3&B6. Knowledge Graph Data Languages (RDF, RDFS, SKOS)</li>

```

Comment:
this seems like a formatting error at the start of the line:3 lines seem to have been munged